README.md

```diff
@@ -55,7 +68,13 @@ If you're using [VS Code](https://code.visualstudio.com/), this template comes w
 
 ## Release your game
 
-This template uses [GitHub workflows](https://docs.github.com/en/actions/using-workflows) to run tests and build releases. See [Workflows](./docs/workflows.md) for more information.
+This template uses [GitHub workflows](https://docs.github.com/en/actions/using-workflows) to run tests and build releases.
+See [Workflows](./docs/workflows.md) for more information.
+
+## (Optional) Improve your compile times
```

Comment:
Hmmm this _could_ be a collapsible section in `Run your game`. It's fine here too, though.

Reply:
Good idea [`0b3eb0b` (#158)](https://github.com/TheBevyFlock/bevy_quickstart/pull/158/commits/0b3eb0b0f2e313dbefa6977a370deb8c1bef1b80)